todo_app/data/trello_items.py

```diff
@@ -41,23 +41,42 @@ def add_item(title):
     params=query
   )
 
+def _get_status_from_list_id(list):
+  if list == os.getenv("TRELLO_FINISHED_LIST_ID"):
+    return "Completed"
+  if list == os.getenv("TRELLO_IN_PROGRESS_LIST"):
+    return "In Progress"
+  if list == os.getenv("TRELLO_NOT_STARTED_LIST_ID"):
+    return "Not Started"
```

Comment:
(Very minor)
This function caused me a quick issue, because I have more than 3 lists on my test board, so I needed to just add a quick:
```python
  else:
    return "Other"
```
At the end.

Now I think you can look at this in two ways, and I don't know which my instinct prefers!

1. We should defensively program - we can then accept the risk of someone adding a fourth list, and we can just have it not show up on our app
2. We want to get a big glaring error message if someone adds a fourth list! It's not been tested, and we don't want to have to support that case

I guess we could combine those, with something like:
3. Defensively program so that it doesn't error, but then add a log message (when we get round to adding those properly!) set at warning/error or something, so that we flag this as a concern someone should be looking into

Like I say, not one I have strong opinions on, but an interesting thought exercise